doc/Developing.md

```diff
@@ -24,7 +24,9 @@
 ## Building the client
 
 1. Clone the repository
-2. Configure your system using the [configuration file](../.config/configuration.winget) in the repository. To run the configuration, use `winget configure .config/configuration.winget` from the project root so relative paths resolve correctly.
+2. Configure your system using the [configuration file](../.config/configuration.winget) in the repository. To run the configuration with VS Community, use `winget configure .config/configuration.winget` from the project root so relative paths resolve correctly.
```

Comment:
nit: I think the Community version would better be in the list alongside the Pro and Enterprise version

- For Community ...
- For Pro ....
- For Enterprise ...

Reply:
Agree and fixed!